llvm/lib/Transforms/Vectorize/VPlanValue.h

```diff
@@ -75,7 +75,7 @@ class VPValue {
 public:
   /// Return the underlying Value attached to this VPValue.
   Value *getUnderlyingValue() { return UnderlyingVal; }
-  const Value *getUnderlyingValue() const { return UnderlyingVal; }
+  Value *getUnderlyingValue() const { return UnderlyingVal; }
```

Comment:
This loses const'ness? Makes the non-const version redundant?

Reply:
Yep, removed, thanks!